From: Jeremy Bicha <jbicha@debian.org>
Date: Mon, 21 Jan 2019 19:32:46 -0500
Subject: translate Debian directories

This is really only useful for Ubuntu
since Debian uses the po-up system
---
 po/POTFILES.in | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

Index: gnome-menus-3.35.3/po/POTFILES.in
===================================================================
--- gnome-menus-3.35.3.orig/po/POTFILES.in
+++ gnome-menus-3.35.3/po/POTFILES.in
@@ -15,3 +15,19 @@ desktop-directories/X-GNOME-Other.direct
 desktop-directories/X-GNOME-Sundry.directory.desktop.in
 desktop-directories/X-GNOME-Utilities.directory.desktop.in
 desktop-directories/X-GNOME-WebApplications.directory.desktop.in
+debian/desktop-files/ActionGames.directory.desktop.in
+debian/desktop-files/AdventureGames.directory.desktop.in
+debian/desktop-files/ArcadeGames.directory.desktop.in
+debian/desktop-files/BlocksGames.directory.desktop.in
+debian/desktop-files/BoardGames.directory.desktop.in
+debian/desktop-files/CardGames.directory.desktop.in
+debian/desktop-files/Debian.directory.in
+debian/desktop-files/GnomeScience.directory.in
+debian/desktop-files/KidsGames.directory.desktop.in
+debian/desktop-files/LogicGames.directory.desktop.in
+debian/desktop-files/RolePlayingGames.directory.desktop.in
+debian/desktop-files/Settings-System.directory.in
+debian/desktop-files/Settings.directory.in
+debian/desktop-files/SimulationGames.directory.desktop.in
+debian/desktop-files/SportsGames.directory.desktop.in
+debian/desktop-files/StrategyGames.directory.desktop.in
